apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -168,7 +168,7 @@ export function InstructorQuestionSettings({
                 <tbody>
                   <tr>
                     <th class="align-middle">
-                      <label id="topic-label" for="topic">Topic</label>
+                      <label id="topic-label" for="topic" class="form-label">Topic</label>
```

Comment:
These changes force things out of alignment.

Before:

<img width="622" alt="Screenshot 2025-03-12 at 14 20 08" src="https://github.com/user-attachments/assets/87c90f0f-b4db-472b-abb3-c4d5ed3ba382" />

After:

<img width="619" alt="Screenshot 2025-03-12 at 14 19 48" src="https://github.com/user-attachments/assets/6d9518d7-f4a0-476b-a347-da1d928cde42" />

Reply:
Updated in f11866b60dec057d9ac57693d3df97713e1448ac to match the compat behaviour (i.e., only change this for labels inside a form-group).

Reply:
Thanks! Continuing discussion here: https://github.com/PrairieLearn/PrairieLearn/pull/11545#discussion_r1992105920